docs/azure/events.md

```diff
@@ -0,0 +1,44 @@
+# Semantic Conventions for Azure Resource Log events
+
+**Status**: [Experimental][DocumentStatus]
+
+This document defines semantic conventions for instrumentations that emit Azure
+Resource Log events.
+
+## Azure Resource Log
+
+### Event details
+
+The event name MUST be `az.resource.log` in the `event.name` property.
+
+### Attributes
+
+<!-- manually added table until body fields can be autogenerated -->
```

Comment:
these are attributes, you can generate them automatically (for `az.resource.log` group)

Reply:
I thought the attributes table would have been autogenerated using the make targets `table-generation` or `attribute-registry-generation` but it didn't work so created manually.

Is there another make target that I should use instead?

Reply:
outside if attribute registry, markdown files are mostly written manually, only tables between 
`<!-- semconv  az.resource.log -->` and `<!-- endsemconv -->` are auto-generated (with `table-generation`) target.

Do you just need to add the comments above and run the target.

Reply:
Nice, thanks you for the help. I've updated to autogenerate the attributes in [0c98f83](https://github.com/open-telemetry/semantic-conventions/pull/1028/commits/0c98f834a9146bfc7ba60795d8a143b1f9b1c3ac)